### ISSUE TITLE
Shimmer not working properly after flutter upgrade

### ISSUE THREAD

**Issue by fajarainul** [hnvn/flutter_shimmer#18]
After I used command `flutter upgrade` the shimmer not working properly on android device (I use genymotion). Any ideas?

Here my `flutter doctor -v`

```
[✓] Flutter (Channel stable, v1.12.13+hotfix.5, on Mac OS X 10.13.6 17G65, locale en-ID)
    • Flutter version 1.12.13+hotfix.5 at /Users/somatech/Documents/tools/flutter
    • Framework revision 27321ebbad (5 weeks ago), 2019-12-10 18:15:01 -0800
    • Engine revision 2994f7e1e6
    • Dart version 2.7.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/somatech/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)
    • All Android licenses accepted.

[!] Xcode - develop for iOS and macOS (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    ✗ Flutter requires a minimum Xcode version of 10.2.0.
      Download the latest version or update via the Mac App Store.
    • CocoaPods version 1.7.5

[✓] Android Studio (version 3.5)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 42.1.1
    • Dart plugin version 191.8593
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b49-5587405)

[!] IntelliJ IDEA Ultimate Edition (version 2019.2.2)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.dev/intellij-setup/#installing-the-plugins

[✓] Connected device (1 available)
    • Samsung • 192.168.56.103:5555 • android-x86 • Android 8.0.0 (API 26)
``` 

Thank you for everything!

**Comment by hnvn** [comment]
Does it happen on the [example](https://github.com/hnvn/flutter_shimmer/tree/master/example) project?

**Comment by fajarainul** [comment]
@hnvn please give a moment, let me try.

**Comment by fajarainul** [comment]
@hnvn **update** : yes it does. 
The shimmer looks like this

![screenshot-2020-01-17_11 17 31 849](https://user-images.githubusercontent.com/6522915/72584004-fb85bc00-391a-11ea-88d8-cb8615459643.png)
![Uploading screenshot-2020-01-17_11.17.33.458.png…]()

**Comment by hnvn** [comment]
Ok, I will check it.

**Comment by hnvn** [comment]
I've just run the example project on Flutter v1.12.13+hotfix.5 but I cannot reproduce your issue

**Comment by fajarainul** [comment]
so, anything wrong with my codes? or tools? anything?

**Comment by fajarainul** [comment]
@hnvn You sure, use latest flutter version? cause I still facing this issue after change to my friend laptop. Thanks

**Comment by hnvn** [comment]
I have an application using this package and I've just updated it to Flutter v1.12.13+hotfix.5 and it's on AppStore now.

**Comment by fajarainul** [comment]
Can you please try it on Android? Cause we face this issue on android device.

**Comment by hnvn** [comment]
Have you fixed your issue yet? I have no idea about your case.  I am still waiting for anyone facing your issue too, but it seems it's your specific issue.

**Comment by fajarainul** [comment]
not yet. 
My team decide to rollback flutter framework to previous version. 
On IOS everything works fine, after upgrade to Flutter's latest version, but on Android error occurred. Like my previous image.

**Comment by fajarainul** [comment]
any update? @hnvn

**Comment by majindageta** [comment]
I'm running flutter 1.12.13+hotfix.7 and not having any issue with this library

**Comment by hnvn** [comment]
@fajarainul , I cannot reproduce your issue at all. Just tested on Android 7 and Flutter v1.12.13+hotfix.8

**Comment by fajarainul** [comment]
Ok, let me try again. Thanks for your response.

**Comment by fajarainul** [comment]
I just tested on real device and emulator (Genymotion).
Using Flutter 1.12.13+hotfix8.
Everything work fine on real device, but in genymotion the problem still occured.
I will close this issue.
Thanks all.
